saskatoon/sitebase/templates/registration/change_password.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'registration/base.html' %}
```

Comment:
what prevented you from being able to reuse a basic form base like https://github.com/LesFruitsDefendus/saskatoon-ng/blob/develop/saskatoon/sitebase/templates/app/forms/base.html ? Can you add a couple screenshots to explain what  your workaround is achieving for displaying form errors?

Reply:
I chose to use the existing `PasswordChangeForm` because it was already there and did most of the heavy lifting.

Workaround was to avoid rendering the fields rendered as text in `ul` elements. Because the text isn't inside any other child element, i couldn't style it with `display: none;`. Wasn't sure how else to go about hiding the arbitrary field names either through templates or form.

here's what it would look like with a simple `{{ form.errors }}`

![20240430_16h10m39s_grim](https://github.com/LesFruitsDefendus/saskatoon-ng/assets/75142042/695f0a3a-e147-49bf-a7a5-4dd3e85e8326)
![20240430_16h12m03s_grim](https://github.com/LesFruitsDefendus/saskatoon-ng/assets/75142042/60eb7089-d326-428d-9c74-b7983e8de501)

Reply:
I think ` {{ form|crispy }} ` should do that for you, see other comment